pkg/cli/environment.go

```diff
@@ -135,9 +152,10 @@ func (s *EnvSettings) EnvVars() map[string]string {
 		"HELM_MAX_HISTORY":       strconv.Itoa(s.MaxHistory),
 
 		// broken, these are populated from helm flags and not kubeconfig.
-		"HELM_KUBECONTEXT":   s.KubeContext,
-		"HELM_KUBETOKEN":     s.KubeToken,
-		"HELM_KUBEAPISERVER": s.KubeAPIServer,
+		"HELM_KUBECONTEXT":            s.KubeContext,
+		"HELM_KUBETOKEN":              s.KubeToken,
+		"HELM_KUBEAPISERVER":          s.KubeAPIServer,
+		"HELM_KUBEAPISERVER_INSECURE": strconv.FormatBool(s.Insecure),
```

Comment:
kubectl does not accept an environment variable for `--insecure-tls-verify`. Why should Helm?